File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.cpp
class Solution {
public:
    bool uniqueOccurrences(vector<int>& arr) {
        unordered_map<int,int> m;
        for(int i=0;i<arr.size();i++){
            m[arr[i]]++; 
        }
        arr.clear();
        for(auto &val:m){
            arr.push_back(val.second);
        }
        for(int i=0;i<arr.size();i++){
            for(int j=i+1;j<arr.size();j++){
                if(arr[i] == arr[j]){
                    return false;
                }
            }
        }
        return true;
        
    }
};

File: 1207-unique-number-of-occurrences/1207-unique-number-of-occurrences.py
class Solution(object):
    def uniqueOccurrences(self, arr):
        """
        :type arr: List[int]
        :rtype: bool
        """
        m = {}
        for number in arr:
            m[number] = arr.count(number)
        mylist = list(m.values())
        return len(mylist) == len(set(mylist))
        